### PR TITLE
fix(challenge): invert icon on light mode

### PR DESCRIPTION
**Pull request by st3iny**
## Before

before

## After
| Light | Dark |
| --- | --- |
| grafik | grafik |

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

File: css/style.css [hunk 1]
```diff
@@ -9,4 +9,8 @@
 
 .totp-form > input[type='tel'] {
 	margin: 0 0 5px 0;
-}
\ No newline at end of file
+}
+
+.two-factor-totp-icon {
+    filter: var(--background-invert-if-bright);
```
Comment (ChristophWurst) [inline css/style.css:15]:
```suggestion
	filter: var(--background-invert-if-bright);
```

**Review by ChristophWurst — DISMISSED** [review]
linter isn't happy

File: css/style.css (hunk 1 above)
Comment (st3iny) [reply css/style.css:15]: Done

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**